### DISCUSSION TITLE
Conversion of CP2K output file using dpdata

### DISCUSSION THREAD

**Discussion by mayankbisht1** [deepmodeling/dpdata#615] [Q&A]
Facing difficulty converting the cp2k output file into compressed data format using dpdata.

**Comment by njzjz** [comment]
FYI @robinzyb

**Comment by robinzyb** [comment]
hello, Could I have your output ? and the script of converting to dpdata, and error messages etc.

**Reply by mayankbisht1** [reply]
I have started to use DeepMD-kit, but the CP2K output I have till now is of low print level, and later I realized that deepMD-kit requires a Medium level. Please guide me on how to convert this using dpdata, or do I have to start over?

**Reply by robinzyb** [reply]
what kind of output, MD or ENERGY_FORCES?

**Reply by mayankbisht1** [reply]
I'm using the output file from CP2K  .out format from which I want to extract frames. ( MD output)

**Reply by robinzyb** [reply]
check this out 
https://github.com/robinzyb/cp2kdata/blob/main/docs/dpdata_plugin.md
https://github.com/robinzyb/cp2kdata/blob/main/docs/output.md#parse-md-outputs

**Reply by robinzyb** [reply]
what types of MD, NVT or NPT? if NVT , there is a way to around it

**Reply by mayankbisht1** [reply]
I performed the simulation under NPT conditions.
